client/components/ThousandSeparatedDecimalInput/ThousandSeparatedDecimalInput.tsx

```diff
@@ -1,8 +1,9 @@
-import React from 'react'
-import * as R from 'ramda'
 import './ThousandSeparatedDecimalInput.scss'
-import { formatDecimal } from '@common/numberFormat'
+import React from 'react'
+
+import { Numbers } from '@core/utils'
 import { acceptableAsDecimal } from '@webapp/utils/numberInput'
+import * as R from 'ramda'
```

Comment:
do we still use R? maybe we can wipe it out in another refactor...

Reply:
We have this:
https://github.com/openforis/fra-platform/projects/1#card-76384309